### ISSUE TITLE
GoAccess addon - https://goaccess.io

### ISSUE THREAD

**Issue by acondura** [docksal/addons#39]
Hi guys,

Firstly, I want to say Docksal is fabulous! I really like it, I love it flexibility, extendability and the infinite ways you can do with it.

I'm already using Docksal with two of my clients, which takes me to my second point. I want to create an addon in order to use the GoAccess log analyzer tool, which I think is an amazing tool:

GoAccess is an open source real-time web log analyzer and interactive viewer that runs in a terminal in *nix systems or through your browser. It provides fast and valuable HTTP statistics for system administrators that require a visual server report on the fly.

I've been reading up on the docs for creating addons and I've created the goaccess folder and its files but it seems I need your help pushing this forward. I've also been reading the code for the other addons but I'm still none the wiser.

Simply put, I can't figure out how do I take the install steps from here:
https://goaccess.io/download#docker
and put them into an addon that can display the log reports either in the cli or in HTML.

How can I share what I've created so far and get your input?

**Comment by acondura** [comment]
Hm... if I'm reading the Docksal docs right, it seems like I need to extend the Docksal images https://docs.docksal.io/stack/extend-images. Going through that now...

**Comment by achekulaev** [comment]
As far as I understand you don't need to extend images, you need to automate those steps by the link by putting them in the custom command and put a new container into docksal.yml

**Comment by acondura** [comment]
Thank you Oleksii, I can't wait to get this working!

**Comment by petertornstrand** [comment]
This would be a very nice feature.
